extensionsGallery.json

```diff
@@ -5632,6 +5632,67 @@
 					  ],
 					  "statistics": [],
 					  "flags": ""
+				},
+				{
+					"extensionId": "111",
+					"extensionName": "couchbase",
+					"displayName": "Couchbase",
+					"shortDescription": "Welcome to the official Visual Studio Code extension for Couchbase! This extension is designed to provide a seamless experience for Couchbase Server users who want to work within the popular Visual Studio Code editor.",
```

Comment:
Update this description to also mention Azure Data Studio instead of Visual Studio Code.